### PR TITLE
LP-2698 Revoke Admin user to delete webinar registration

### PR DESCRIPTION
**Pull request by aliassad**

### CODE REVIEW

File: openedx/adg/lms/webinars/admin.py [hunk 1]
```diff
@@ -187,6 +187,26 @@ def get_queryset(self, request):
...
+        includes webinar registrations. Since webinar registration’s delete permission is revoked from the admin site,
+        the admin delete view show a permission error and does not allow to proceed with the cancellation of the
+        webinar. To bypass the permission error, we are overriding this method.
+
+        Note: deleting (cancelling) a webinar does not delete its corresponding webinar registrations, we only need the
+        permission so that the webinar can be cancelled successfully
+        """
+        deleted_objects, model_count, perms_needed, protected = super().get_deleted_objects(
+                                                                                        objs, request
+                                                                                        )  # pylint: unused-variable
```
Comment (harris-irfan) [inline openedx/adg/lms/webinars/admin.py:207]: Please follow [edx's rules of breaking long lines](https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#breaking-long-lines). This is not clean.

File: openedx/adg/lms/webinars/tests/test_admin.py [hunk 2]
```diff
@@ -268,3 +268,14 @@ def test_registration_webinar_admin_delete_permission():
...
+
+
+@pytest.mark.django_db
+def test_get_deleted_objects_webinar_admin(webinar_admin_instance, request, webinar):
+    """
+    Test that get_deleted_objects for webinar admin returns an empty set of permissions
+    """
+    request.user = UserFactory()
+    deleted_objects, model_count, perms_needed, protected = webinar_admin_instance.get_deleted_objects([webinar],
+                                                                                                       request)
```
Comment (harris-irfan) [inline openedx/adg/lms/webinars/tests/test_admin.py:280]: Same as above

**Review by harris-irfan — DISMISSED** [review]
In case it is already merged in develop, we can't do much, but leaving comments for future reference.

File: openedx/adg/lms/webinars/admin.py (hunk 1 above)
Comment (aliassad) [reply openedx/adg/lms/webinars/admin.py]: Done

File: openedx/adg/lms/webinars/tests/test_admin.py (hunk 2 above)
Comment (aliassad) [reply openedx/adg/lms/webinars/tests/test_admin.py]: Done

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - Edx
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/1932/workflows/workflows_id/jobs/1932/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/master...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/webinars/admin.py (100%)
openedx/adg/lms/webinars/tests/test_admin.py (100%)
-------------
Total:   8 lines
Missing: 0 lines
Coverage: 100%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - Edx
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/1935/workflows/workflows_id/jobs/1935/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/master...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/webinars/admin.py (100%)
openedx/adg/lms/webinars/tests/test_admin.py (100%)
-------------
Total:   8 lines
Missing: 0 lines
Coverage: 100%
-------------
```

**Review by harris-irfan — APPROVED** [review]

**Pull request merged**